08-transport.md

```diff
@@ -459,7 +451,7 @@ construction, and `16 bytes` for a final authenticating tag.
     abort the connection attempt.
 
 
-  * `rs = decryptWithAD(temp_k, n, h, c)`
+  * `rs = decryptWithAD(temp_k, 2, h, c)`
```

Comment:
(commenting here as I can't on the line below) 

The modification on line 406 is incorrect. The actual error is on line 459, the receiver should mix the  _ciphertext_ into the handshake rather than the decrypted material (the pubkey).